kube/deployment-dev.yml

```diff
@@ -79,7 +79,7 @@ spec:
           - --verbose
 
       - name: proxy
-        image: quay.io/ukhomeofficedigital/dq-nginx-proxy-redirect:dev-72
+        image: quay.io/ukhomeofficedigital/dq-nginx-proxy-redirect:b224
```

Comment:
This b tag is for prod isnt? shouldnt it be a prefix with dev for this one?

Reply:
DEV should have "b" because we want to have same version between not-prod and prod. DEV-72 has got EXTRAORDINARY vulnerabilities that is very very old. Hope this clarifies Musti